README.md

```diff
@@ -11,7 +11,7 @@ By default, this library will look for `packs` in the folder `packs/*/package.ym
 ```yml
 pack_paths:
   - "{packs,utilities,deprecated}/*" # packs with multiple roots!
-  - "{packs,utilities,deprecated}/*/*" # nested packs!
+  - "{packs,utilities,deprecated}/**/*" # nested packs!
```

Comment:
Thank you for the suggestion.

This change will enable unlimited levels of nesting which can be a performance problem for large code bases. For this reason, we want to demonstrate one level of nesting in the docs.

If you disagree, let me know.